backend/src/db/migrations/20230909170408_make_topic_id_of_meditation_entries.ts

```diff
@@ -0,0 +1,23 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'meditation_entries';
+
+const ColumnName = {
+  TOPIC_ID: 'topic_id',
+} as const;
+
+function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(TABLE_NAME, (table) => {
+    table.integer(ColumnName.TOPIC_ID).nullable().alter();
+  });
+}
+
+async function down(knex: Knex): Promise<void> {
+  await knex(TABLE_NAME).del().where(ColumnName.TOPIC_ID, null);
+
+  await knex.schema.alterTable(TABLE_NAME, (table) => {
+    table.integer(ColumnName.TOPIC_ID).notNullable().alter();
```

Comment:
It might fail if there are records with nullable TOPIC_ID

Reply:
Just in line 16 I deleted all records, where `topic_id` is null as @Liza-Veis suggested [here](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/213#discussion_r1320758612)) It can't fail

Reply:
I should have checked whether you understand this and not just follow the advice)